### ISSUE TITLE
Is it possible to get the validation set used in fit?

### ISSUE THREAD

**Issue by nelson-liu** [fchollet/keras#2596]
Hi,
I'm wondering whether it's possible to access the validation set used in `fit()`. I read that `The validation data is picked as the last 10% (for instance, if validation_split=0.9) of the input.` (https://github.com/fchollet/keras/issues/597), could i just slice off the last 10% and that would be my validation set?

Thanks!

**Comment by joelthchao** [comment]
You can split train/val set by yourself and feed into `fit()` likes follow:
```python
model.fit(X_train, Y_train, validation_data=(X_val, Y_val))
```
